java/src/main/java/com/xebia/functional/xef/java/auto/AIScope.java

```diff
@@ -4,27 +4,24 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.module.jsonSchema.JsonSchema;
 import com.fasterxml.jackson.module.jsonSchema.JsonSchemaGenerator;
+import com.xebia.functional.xef.auto.AIRuntime;
 import com.xebia.functional.xef.auto.CoreAIScope;
 import com.xebia.functional.xef.embeddings.Embeddings;
-import com.xebia.functional.xef.embeddings.OpenAIEmbeddings;
-import com.xebia.functional.xef.env.OpenAIConfig;
-import com.xebia.functional.xef.llm.openai.KtorOpenAIClient;
-import com.xebia.functional.xef.llm.openai.LLMModel;
-import com.xebia.functional.xef.llm.openai.CFunction;
-import com.xebia.functional.xef.llm.openai.images.ImageGenerationUrl;
-import com.xebia.functional.xef.llm.openai.images.ImagesGenerationResponse;
+import com.xebia.functional.xef.llm.AIClient;
+import com.xebia.functional.xef.llm.LLM;
+import com.xebia.functional.xef.llm.LLMModel;
+import com.xebia.functional.xef.llm.models.functions.CFunction;
+import com.xebia.functional.xef.llm.models.images.ImageGenerationUrl;
+import com.xebia.functional.xef.llm.models.images.ImagesGenerationResponse;
+import com.xebia.functional.xef.llm.openai.OpenAIClient;
+import com.xebia.functional.xef.pdf.PDFLoaderKt;
 import com.xebia.functional.xef.textsplitters.TextSplitter;
 import com.xebia.functional.xef.vectorstores.LocalVectorStore;
 import com.xebia.functional.xef.vectorstores.VectorStore;
 import kotlin.collections.CollectionsKt;
 import kotlin.coroutines.Continuation;
 import kotlin.jvm.functions.Function1;
-import com.xebia.functional.xef.pdf.PDFLoaderKt;
-import kotlinx.coroutines.CoroutineScope;
-import kotlinx.coroutines.CoroutineStart;
-import kotlinx.coroutines.ExecutorsKt;
-import kotlinx.coroutines.JobKt;
-import kotlinx.coroutines.CoroutineScopeKt;
+import kotlinx.coroutines.*;
```

Comment:
No start imports.

Reply:
Removed in https://github.com/xebia-functional/xef/pull/196/commits/f48bca33c96eaf6d389aa5e4371deb43366532bc